src/pages/home/report/ReportActionsView.js

```diff
@@ -125,9 +125,10 @@ class ReportActionsView extends React.Component {
             Report.updateLastReadActionID(this.props.reportID);
         });
 
-        // If the reportID is not found then we have either not loaded this chat or the user is unable to access it.
+        // If the reportID is not found or there is no last message timestamp then
+        // we have either not loaded this chat or the user is unable to access it.
         // We will attempt to fetch it and redirect if still not accessible.
-        if (!this.props.report.reportID) {
+        if (!this.props.report.reportID || !this.props.report.lastMessageTimestamp) {
```

Comment:
I've never liked this existing logic because I think it's really unclear why how we save a report that has no `reportID` into Onyx. 

This change makes it even more unclear because we also have a situation where the `lastMessageTimestamp` doesn't exist. 

Can anyone answer how these reports get added and why are they missing these details?

Ultimately, it seems like the only case where we should have to "fetch" a report is if we get a Pusher update for one and it doesn't exist. I can't quite think of why else we'd need this - but also not sure how our API changes would handle this.

Basically - this is not a great design and while this crash seems bad - it's important to solve this in a way that makes sense (and probably do this internally so we can consider backend changes).